community_images/common/orchestrator/registry_helper.py

```diff
@@ -92,7 +92,8 @@ def get_latest_tag_with_digest(self, account, repo, search_str):
         search_str = re.compile(search_str)
         tags = filter(lambda tag: search_str.search(tag["name"]), tags)
         tags = list(filter(
-            lambda tag: "rfstub" not in tag["name"] and tag["tag_last_pushed"],
+            lambda tag: "rfstub" not in tag["name"] and tag["tag_last_pushed"] and\
+                any(image.get('os', '') == 'linux' for image in tag.get('images', [])),
```

Comment:
added this check since there are few tags for which image is not available on linux so restricting the search of tags available on linux